packages/web-components/src/components/cbp-banner/cbp-banner.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component, Host, Prop, Element,  h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-banner',
+  styleUrl: 'cbp-banner.scss',
+})
+export class CbpBanner {
+
+  @Element() host: HTMLElement;
+
+  @Prop() color: string = 'infoBanner'; //intended to be used for different statuses (error, warning, etc)
+
+  handleDismiss() {
+    this.host.setAttribute('hidden', '');
+    //call event emitter for app to piggy back (need Event, EventEmitter ?)
+  }
+
+  render() {
+    return (
+      <Host aria-color={this.color}>
+        <div class='iconContainer'>
+          <cbp-icon  
+            name='exclamation-circle'
+            color='var(--cbp-color-text-lighter)'
+            size='3rem'
+            />
+        </div>
+        <div class='textContainer'>
```

Comment:
Same issue with class naming. This is very generic and could encounter collisions.